mirrors.yml

```diff
@@ -1,15 +1,12 @@
 origin:
     desc: AOSC main repository (hosted by BlueT, and National Chi Nan University)
```

Comment:
no longer bluet

Reply:
Is it still hosted by National Chi Nan University? If no, what is it?

Reply:
`(sponsored by xTom.com)`

Reply:
only sponsored by xTom.com ?